examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -493,7 +493,7 @@ int seq_run(const traccc::opts::detector& detector_opts,
                             const_proxy_type>{
                         vecmem::get_data(measurements_per_event),
                         vecmem::get_data(measurements_per_event_cuda)}};
-            compare_track_candidates(
+            compare_resolved_track_candidates(
```

Comment:
Hmm... The only difference between `compare_track_candidates` and `compare_resolved_track_candidates` is the string printout that they make.

For clear readability this update should be done indeed. But I don't see how this change should affect the calculated efficiencies. 😕 (For context, that's the discussion that we started from with Beomki.) Since both comparator functors operate with the same measurements. 🤔

Reply:
You need to check the codes in my DM again and understand why it did not work for me